docs/release/1.15/release-note-1.15.0.md

```diff
@@ -0,0 +1,42 @@
+# Release Note 1.15.0
+
+## ONE Compiler
+
+### Compiler Frontend
+
+- Support more Ops for quantization
+- Fix `record-minmax` tool for bool type, NaN values
+- Fix `one-cmds` test scripts
+- Remove `stdex` module
+- `arser` supports short option
+
+
+## ONE Runtime
+
+### Runtime backend operation supports more operations and types
```

Comment:
I'm not sure. There might be `Runtime backend operation` module or sth if I am wrong.
```suggestion
### Runtime backend supports more operations and types
```